store/sqlstore/upgrade.go

```diff
@@ -180,6 +190,60 @@ func upgradeDatabase(sqlStore SqlStore, currentModelVersionString string) error
 	return nil
 }
 
+func decideMaxExecutionTime(sqlStore SqlStore) int64 {
+	result := MAX_TIMEOUT_EXECUTION_TIME_MYSQL
+	if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
+		result = MAX_TIMEOUT_EXECUTION_TIME_POSTGRES
+	}
+	return result
+}
+
+func getAndSetMaxExecutionTime(sqlStore SqlStore, value int64) int64 {
+	var err error
+	var currentExecutionTimeout int64
+
+	if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
+		err = sqlStore.GetMaster().QueryRow("SHOW STATEMENT_TIMEOUT").Scan(&currentExecutionTimeout)
+		if err != nil {
+			mlog.Critical("Failed to read statement_timeout parameter from postgres", mlog.Err(err))
+			time.Sleep(time.Second)
+			os.Exit(EXIT_INCREASING_EXECUTION_TIMEOUT_POSTGRES)
+		}
+	} else if sqlStore.DriverName() == model.DATABASE_DRIVER_MYSQL {
+		var paramName string
+		err = sqlStore.GetMaster().QueryRow("SHOW VARIABLES LIKE 'WAIT_TIMEOUT'").Scan(&paramName, &currentExecutionTimeout)
+		if err != nil {
+			mlog.Critical("Failed to read max_execution_time parameter from mysql", mlog.Err(err))
+			time.Sleep(time.Second)
+			os.Exit(EXIT_INCREASING_EXECUTION_TIMEOUT_MYSQL)
+		}
+	}
+
+	setMaxExecutionTime(sqlStore, value)
+
+	return currentExecutionTimeout
+}
+
+func setMaxExecutionTime(sqlStore SqlStore, value int64) {
+	var err error
+	if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
+		_, err = sqlStore.GetMaster().Exec(fmt.Sprintf("SET STATEMENT_TIMEOUT = %d", value))
```

Comment:
Probably is a better option this:
```suggestion
		_, err = sqlStore.GetMaster().Exec("SET STATEMENT_TIMEOUT = $1", value)
```